content/blog/managing-aws-credentials-on-cicd/index.md

```diff
@@ -0,0 +1,231 @@
+
+The approach includes the following:
+
+- Create a dedicated IAM User for your CI/CD
+- Provide the IAM User’s credentials to your CI/CD system
+- Automate rotating and revoking that user’s credentials
+- During deployments, exchange the IAM User credentials for more privileged IAM Role credentials
+
+The process may sound complicated, but we’ll cover each step in detail.  We’ll point out additional ways to enhance your security, such as using Pulumi’s support for client-side encryption to limit access to sensitive information.
+
+## Create a dedicated IAM User for your CI/CD
+
+The first step for securely automating CI/CD on AWS is to create a dedicated IAM User for use in your CI/CD pipelines.
+
+By default, a new IAM User does not have any permissions associated with it, meaning it can only authenticate itself with its access key and secret. Let’s walk through how to grant this IAM User the permissions necessary to update your cloud infrastructure later. For now, we limit this user to have as little access as possible.
+
+The following code snippet shows how to create an IAM User specifically for your CI/CD jobs. 
+
+```ts
+const aws = require("@pulumi/aws");
+
+const role = new aws.iam.Role("myrole", {
+    assumeRolePolicy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [{
+            Action: "sts:AssumeRole",
+            Principal: {
+                Service: "ec2.amazonaws.com"
+            },
+            Effect: "Allow",
+            Sid: ""
+        }]
+    })
+});
+
+const rolePolicy = new aws.iam.RolePolicy("myrolepolicy", {
+    role: role,
+    policy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [{
+            Action: [ "ec2:Describe*" ],
+            Effect: "Allow",
+            Resource: "*"
+        }]
+    })
+});
+
+const policy = new aws.iam.Policy("mypolicy", {
+    policy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [{
+            Action: [
+              "ec2:Describe*"
+            ],
+            Effect: "Allow",
+            Resource: "*"
+        }]
+    })
+});
+
+const rolePolicyAttachment = new aws.iam.RolePolicyAttachment("myrolepolicyattachment", {
+    role: role,
+    policyArn: policy.arn
+});
+
+const user = new aws.iam.User("myuser");
+
+const group = new aws.iam.Group("mygroup");
+
+const policyAttachment = new aws.iam.PolicyAttachment("mypolicyattachment", {
+    users: [user],
+    groups: [group],
+    roles: [role],
+    policyArn: policy.arn
+});
+```
+
+### Isolating with Different Accounts
+
+A common and **recommended** practice is to use multiple AWS accounts to isolate resources and users from one another. For example, if your testing and production environments are in different AWS accounts, it would be difficult for a severe bug in the testing environment to harm your customer data.
```

Comment:
Nit: The emphasis on recommended seems out of place. Is there a particular white paper or something we should be linking to?